doc/ctest_user_manual.md

```diff
@@ -73,6 +73,16 @@ The execution order of the tests in a test suite is not guaranteed. Tests are ex
 
 The `failedTestCount` argument for `CTEST_RUN_TEST_SUITE` is optional. If specified, the number of failed tests will be summed up in the `failedTestCount` variable, that is passed as argument.
 
+## Leak detection (VLD)
+
+When a test is compiled using [Visual Leak Detector](https://github.com/KindDragon/vld), `CTest` will check if there were any memory leaks and report them as test failures.
+
+```c
+CTEST_RUN_TEST_SUITE_WITH_LEAK_CHECK_RETRIES(suiteName{,failedTestCount});
+```
+
+In case a test has some possible memory leaks that are cleaned up asynchronously, leak checks with retries can be enabled by using `CTEST_RUN_TEST_SUITE_WITH_LEAK_CHECK_RETRIES` instead.
```

Comment:
>leak checks with retries can be enabled by using `CTEST_RUN_TEST_SUITE_WITH_LEAK_CHECK_RETRIES` instead. [](start = 82, length = 104)

Consider explaining briefly what that means, i.e. "Leak check retries mean that if a leak is detected at the end of the test, ctest will retry for <insert here how much time> the leak check. If during the time, a leak check observes that no leaks are found, ctest reports the test suite as successful. If leaks are detected over and over again and the time elapses, the test suite is reported as failed.